ui/raidboss/data/05-shb/raid/e10s.js

```diff
@@ -185,9 +185,41 @@ export default {
       netRegexDe: NetRegexes.startsUsing({ source: 'Schattenkönig', id: '5B2D', capture: false }),
       netRegexFr: NetRegexes.startsUsing({ source: 'Roi De L\'Ombre', id: '5B2D', capture: false }),
       netRegexJa: NetRegexes.startsUsing({ source: '影の王', id: '5B2D', capture: false }),
-      alertText: (data, _, output) => output.text(),
+      durationSeconds: (data) => data.gigaSlashCleaveDebuffDuration,
+      alertText: (data, _, output) => {
+        let ret = output.default;
+        switch (data.gigaSlashCleaveDebuffId) {
+        case '973':
+          ret = output.westCleave;
+          break;
+        case '974':
+          ret = output.eastCleave;
+          break;
+        case '975':
+          ret = output.northCleave;
+          break;
+        case '976':
+          ret = output.southCleave;
+          break;
+        }
+        delete data.gigaSlashCleaveDebuffId;
+        delete data.gigaSlashCleaveDebuffDuration;
+        return ret();
+      },
       outputStrings: {
-        text: {
+        northCleave: {
+          en: 'North Cleave',
```

Comment:
Sorry for the second suggestion here, but I am a little bit worried about the default text being so similar to the proper text.  If I was used to hearing "west cleave" and then I heard "left cleave" I might think those are the same thing.  What about saying "north shadow" and "left cleave" to differentiate?  Or, am I overthinking this?

Reply:
Honestly, I would just put the cardinal direction. It’s not like you have to position them very well?

Reply:
So this?
```javascript
      outputStrings: {
        north: {
          en: 'North',
        },
        south: {
          en: 'South',
        },
        east: {
          en: 'East',
        },
        west: {
          en: 'West',
        },
        default: {
          en: 'Left Cleave',
          de: 'Linker Cleave',
          ko: '왼쪽 장판',
        },
      },
```

Reply:
Yeah, that seems good.  Sorry for the small quibble.  I'm just wary of having callouts that might be confusing.

Reply:
The `default` outputString might need to be renamed as well? I'm not sure if that's explanatory enough in the config screen. Maybe it should be `noShadow` instead?

Reply:
This is resolved. I also renamed `default` to `noShadow` for clarity.

Reply:
Should this just always call out left cleave/right cleave + your direction, like two separate triggers?
It feels weird if someone is doing callouts based off cactbot if they have "Right Cleave" for one, and "North" for another.